### PR TITLE
add error explanation

### PR DESCRIPTION
**Pull request by rlmoser99**
Resolves Issue #83

### CODE REVIEW

File: spec/16a_caesar_breaker_spec.rb [hunk 1]
```diff
@@ -85,14 +85,17 @@
...
+    # allowing File.open to raise an error 'Errno::ENOENT'. This error means
+    # that no such file or directory could be found.
     context 'when rescuing an error' do
       before do
         allow(File).to receive(:open).and_raise(Errno::ENOENT)
         allow(phrase).to receive(:puts).twice
       end
 
+      # When an error is rescued, the method will not raise an error.
+      # Therefore you test the conditions that happens if an error was rescued.
```
Comment (msespos) [inline spec/16a_caesar_breaker_spec.rb:98]: "that **would occur** if an error **were** rescued.

File: spec_answers/16a_caesar_answer.rb [hunk 2]
```diff
@@ -102,14 +102,17 @@
...
+    # allowing File.open to raise an error 'Errno::ENOENT'. This error means
+    # that no such file or directory could be found.
     context 'when rescuing an error' do
       before do
         allow(File).to receive(:open).and_raise(Errno::ENOENT)
         allow(phrase).to receive(:puts).twice
       end
 
+      # When an error is rescued, the method will not raise an error.
+      # Therefore you test the conditions that happens if an error was rescued.
```
Comment (msespos) [inline spec_answers/16a_caesar_answer.rb:115]: that **would occur** if an error **were** rescued.

File: spec/16a_caesar_breaker_spec.rb [hunk 3]
```diff
@@ -85,14 +85,17 @@
       end
     end
 
-    # When a method rescues an error, the method will still not raise an error.
-    # Therefore you test the conditions that happens if an error was rescued.
+    # Let's simulate an error occuring during #save_decrypted_messages by
+    # allowing File.open to raise an error 'Errno::ENOENT'. This error means
```
Comment (msespos) [inline spec/16a_caesar_breaker_spec.rb:89]: to raise **the** error

File: spec_answers/16a_caesar_answer.rb [hunk 4]
```diff
@@ -102,14 +102,17 @@
       end
     end
 
-    # When a method rescues an error, the method will still not raise an error.
-    # Therefore you test the conditions that happens if an error was rescued.
+    # Let's simulate an error occuring during #save_decrypted_messages by
+    # allowing File.open to raise an error 'Errno::ENOENT'. This error means
```
Comment (msespos) [inline spec_answers/16a_caesar_answer.rb:106]: to raise **the** error

File: spec/16a_caesar_breaker_spec.rb (hunk 1 above)
Comment (rlmoser99) [reply spec/16a_caesar_breaker_spec.rb]: Wording is updated.

File: spec/16a_caesar_breaker_spec.rb (hunk 3 above)
Comment (rlmoser99) [reply spec/16a_caesar_breaker_spec.rb]: Wording is updated (in both places).

**Pull request merged**